File: CS/CustomMemoEditControl/Form1.cs
using System;
using System.Collections.Generic;
using System.ComponentModel;
using System.Data;
using System.Drawing;
using System.Text;
using System.Windows.Forms;

namespace CustomControl {
    public partial class Form1 : Form {
        public Form1() {
            DevExpress.XtraSpellChecker.Native.SpellCheckTextControllersManager.Default.RegisterClass(typeof(CustomControl.MyMemoEdit), typeof(DevExpress.XtraSpellChecker.Native.SimpleTextEditTextController));
            DevExpress.XtraSpellChecker.Native.SpellCheckTextBoxBaseFinderManager.Default.RegisterClass(typeof(CustomControl.MyMemoEdit), typeof(DevExpress.XtraSpellChecker.Native.MemoEditTextBoxFinder));

            InitializeComponent();
        }

        private void Form1_Load(object sender, EventArgs e) {
            myMemoEdit1.Text = "The quik onyx gobleen jumps over the laizy dvarf";
            myMemoEdit1.DeselectAll();

            memoEdit1.Text = "The quik onyx gobleen jumps over the laizy dvarf";
        }


    }
}

File: VB/CustomMemoEditControl/Form1.vb
Imports Microsoft.VisualBasic
Imports System
Imports System.Collections.Generic
Imports System.ComponentModel
Imports System.Data
Imports System.Drawing
Imports System.Text
Imports System.Windows.Forms

Namespace CustomControl
	Partial Public Class Form1
		Inherits Form
		Public Sub New()
			DevExpress.XtraSpellChecker.Native.SpellCheckTextControllersManager.Default.RegisterClass(GetType(CustomControl.MyMemoEdit), GetType(DevExpress.XtraSpellChecker.Native.SimpleTextEditTextController))
			DevExpress.XtraSpellChecker.Native.SpellCheckTextBoxBaseFinderManager.Default.RegisterClass(GetType(CustomControl.MyMemoEdit), GetType(DevExpress.XtraSpellChecker.Native.MemoEditTextBoxFinder))

			InitializeComponent()
		End Sub

		Private Sub Form1_Load(ByVal sender As Object, ByVal e As EventArgs) Handles MyBase.Load
			myMemoEdit1.Text = "The quik onyx gobleen jumps over the laizy dvarf"
			myMemoEdit1.DeselectAll()

			memoEdit1.Text = "The quik onyx gobleen jumps over the laizy dvarf"
		End Sub


	End Class
End Namespace